addon/components/notification-message.js

```diff
@@ -17,6 +17,11 @@ export default Ember.Component.extend({
         }
     }.property('notification.type'),
 
+    // Apply the clear animation duration rule inline
+    notificationClearDuration: function() {
+        return '-webkit-animation-duration: '+this.get('notification.clearDuration')+'ms; animation-duration: '+this.get('notification.clearDuration')+'ms';
```

Comment:
because you are in Ember you can make use of .fmt() on a string http://emberjs.com/api/classes/Ember.String.html#method_fmt